ansible-protos/runtime.proto

```diff
@@ -6,12 +6,14 @@ message RuntimeData {
 		STUDENT_STOPPED = 2;
 		TELEOP = 3;
 		AUTO = 4;
+		ESTOP = 5;
 	}
 	message SensorData {
 		string device_type = 1;
 		string device_name = 2;
 		float value = 3;
-		int64 uid = 4;
+		uint64 uid = 4;
+		uint32 int_type = 5; 
```

Comment:
What is "int_type"? It's unclear what this field does, and it also doesn't seem to be used anywhere